online_meetings/report.py

```diff
@@ -21,6 +21,26 @@
 DEFAULT_SLEEP_TIME = ENV.get('DEFAULT_SLEEP_TIME', 10)
 
 
+def to_seconds(duration: str) -> int:
+    """Returns a value of a string duration in seconds
+       This string is expected in HH:MM:SS or MM:SS or SS
```

Comment:
Have you seen values in all these time formats?

Reply:
Right. The API returns duration value in that format

Reply:
Yes, all of the values appear to be in this format. I see some meetings lasting exactly 24 hours but nothing longer. It looks like Zoom cuts off at that and there's nothing beyond.